src/pages/Search.tsx

```diff
@@ -62,9 +62,9 @@ const validationSchema = Yup.object()
 const emptyFindRequest: FindRequest = {
   createdMin: '',
   createdMax: '',
-  donorName: '',
+  donorNames: [],
```

Comment:
If the donor name/external name field is left blank, will the fields in the FindRequest be _empty_ or _null_? Because those will have different effects.

Reply:
if not been populated by the user, they will be empty, should they be null in that case ?

Reply:
Yes they should be null